index.html

```diff
@@ -68,6 +68,174 @@ <h1 id="header-first" class="primary-header-text color-deep-blue">Hey there. I
+                    <ul class="category-list">
+                        <li class="category-items">Ruby On Rails</li>
+                        <li class="category-items">CSS</li>
+                        <li class="category-items">JavaScript</li>
+                        <li class="category-items">html</li>
+                    </ul>
+                    <button class="button text-center" type="button">See Project</button>
+                </div>
+            </div>
+            <div class="portfolio-grid-items">
+                <div class="image-container">
+                    <img src="./assets/placeholders/dummy.png" alt="Image Place Holder">
+                </div>
+                <div class="project-description text-center">
+                    <h2 class="title-header">Multi-Post Stories Gain+Glory</h2>
+                    <ul class="category-list">
+                        <li class="category-items">Ruby On Rails</li>
+                        <li class="category-items">CSS</li>
+                        <li class="category-items">JavaScript</li>
+                        <li class="category-items">html</li>
+                    </ul>
+                    <button class="button text-center" type="button">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section id="about-me" class="section-basic about-me-section-bg text-center">
+        <div class="section-main-content-margin text-center">
+            <h1 class="primary-header-text color-deep-blue portfolio-heading">About me</h1>
+        </div>
+        <div class="section-main-content-margin text-center">
+            <p class="heading-supporting-text color-deep-blue">
+                I can help you build a product , feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="section-main-content-margin text-center">
+            <button class="button long-btn text-center" type="button">Get My Resume</button>
+        </div>
+    </section>
+
+    <section id="skills-list" class="section-basic text-center">
+        <div class="section-main-content-margin text-center css-grid">
+            <div class="skills-grid-items">
+                <div class="image-container-skills">
+                    <div class="skill-icons">
+                        <img class="skill-img" src="./assets/icons/skills/icon-01.png" alt="Skill Icon">
+                    </div>
+                </div>
+                <div class="skill-desc text-center">
+                    <h2 class="title-header">Languages</h2>
+                    <ul class="skill-list">
+                        <li class="skill-items">JavaScript</li>
+                        <li class="skill-items">Ruby</li>
+                        <li class="skill-items">HTML</li>
```

Comment:
@AnselemOdims Hi, Thanks for the suggestion. But ### I am a little confused here. 
- You referred to the ```ul``` and ```li``` here, but in the snippet of the ``` index.html ``` I can ``` ul ``` and ``` li ``` already is there.
- Can you please elaborate on the need so that I can understand it clearly?

Reply:
@AnselemOdims Hi, please guide me with my query so that I can start and finish the changes to get approved.
- Please check my previous comment for my queries.
### Thanks.